docs/sources/operators-guide/architecture/components/ingester.md

```diff
@@ -42,10 +42,11 @@ Write de-amplification is the main source of Mimir's low total cost of ownership
 ## Ingesters failure and data loss
 
 If an ingester process crashes or exits abruptly, all the in-memory series that have not yet been uploaded to the long-term storage could be lost.
-There are two primary ways to mitigate this failure mode:
+There are three ways to mitigate this failure mode:
```

Comment:
I would just say "There are the following ways" instead of stating a number, because this number is bound to get outdated when people edit the list in the future

Reply:
Done in https://github.com/grafana/mimir/pull/2183/commits/cb6b707fd21bb6df57be7c8412e17ce33ce672d0